### PR TITLE
DA-1069 add update_done event to help save annotation

### PR DESCRIPTION
**Pull request by rayhu**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
added update_done event to better saving

* **What is the current behavior?** (You can also link to an open issue here)
there are multiple modification events

* **What is the new behavior (if this is a feature change)?**
there will be only one update_done event

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
no, only new event is added.

* **Other information**:
removed excessive console log of mouse move

### CODE REVIEW

File: src/manipulators/moveAllHandles.js [hunk 1]
```diff
@@ -210,6 +210,18 @@ function _upOrEndHandler(
...
+  const modifiedEventData = {
+    toolName,
+    toolType: toolName, // Deprecation notice: toolType will be replaced by toolName
+    element,
+    measurementData: annotation,
+  };
+
+  external.cornerstone.updateImage(element);
+  triggerEvent(element, EVENTS.MEASUREMENT_UPDATE_DONE, modifiedEventData);
+  console.log('done updating move all handles');
```
Comment (mohghaderi) [inline src/manipulators/moveAllHandles.js:223]: remove the log if not needed. getLogger must be used instead if needed

File: src/manipulators/moveHandle.js [hunk 2]
```diff
@@ -239,6 +239,18 @@ function _upOrEndHandler(
...
+  const modifiedEventData = {
+    toolName,
+    toolType: toolName, // Deprecation notice: toolType will be replaced by toolName
+    element,
+    measurementData: annotation,
+  };
+
+  external.cornerstone.updateImage(element);
+  triggerEvent(element, EVENTS.MEASUREMENT_UPDATE_DONE, modifiedEventData);
+  console.log('done updating move one handle');
```
Comment (mohghaderi) [inline src/manipulators/moveHandle.js:252]: remove the log if not needed. getLogger must be used instead if needed

**Review by mohghaderi — CHANGES_REQUESTED** [review]

File: src/manipulators/moveAllHandles.js (hunk 1 above)
Comment (rayhu) [reply src/manipulators/moveAllHandles.js:223]: done

File: src/manipulators/moveHandle.js (hunk 2 above)
Comment (rayhu) [reply src/manipulators/moveHandle.js:252]: done

**Review by mohghaderi — APPROVED** [review]

**Pull request merged**